Plugins/BTCPayServer.RockstarDev.Plugins.TransactionCounter/Controllers/PublicCounterController.cs

```diff
@@ -0,0 +1,108 @@
+using System.Linq;
+using System.Threading.Tasks;
+using BTCPayServer.Client.Models;
+using BTCPayServer.Data;
+using BTCPayServer.Models;
+using BTCPayServer.RockstarDev.Plugins.TransactionCounter.ViewModels;
+using BTCPayServer.Services;
+using BTCPayServer.Services.Invoices;
+using BTCPayServer.Services.Stores;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace BTCPayServer.RockstarDev.Plugins.TransactionCounter.Controllers;
+
+[AllowAnonymous]
+[Route("server/stores/")]
```

Comment:
```suggestion
[Route("txcounter/")]
```

Let's switch to `/txcounter/` path, no need to prefix it with `/server/stores/`.

And then for the other two paths just have:
- `html` for HTML view
- `api` for API call